static/window-template.tmpl

```diff
@@ -0,0 +1,333 @@
+<!doctype html>
+<html lang="en-us">
+    <head>
+        <title>CPython</title>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+

```

Comment:
i think you need to add that ` <meta name="mobile-web-app-capable" content="yes">` for apple devices here.
but i don't have any to verify that say.

Reply:
oo

Reply:
ill add tht in the new verson of tht template